tests/sles4sap/trento/test_hana_restore_stopped.pm

```diff
@@ -37,9 +37,8 @@ sub run {
     my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
     # vmhana01 hard-coded in place of the generic Ansible filter from $primary_host.
     # The Ansible generic filter is only valid for Ansible, here it is crm.
-    qesap_ansible_cmd(cmd => "sudo crm resource refresh rsc_SAPHana_HDB_HDB00 vmhana01",
-        provider => $prov,
-        filter => $primary_host);
+    my $rsc_name = get_var('USE_SAP_HANA_SR_ANGI') ? "SAPHanaCtl" : "SAPHana";
+    qesap_ansible_cmd(cmd => "sudo crm resource refresh rsc_${rsc_name}_HDB_HDB00 vmhana01", provider => $prov, filter => $primary_host);
```

Comment:
I don't think this module is used on the on prem HanaSR test. Perhaps we can leave this one out, but let's confirm with @mpagot as it's trento test code.

Reply:
In future, they will move there. I look for every place where SAPHana resource is mentioned and prepare for this future change

Reply:
Trento test module can be ignored. Can be leaved out from this PR or can be included but in this case we should try to run a VR for it, that is no more so easy to do.

Reply:
OK, I will revert it to the original condition and just left a comment there about future change needed after SAPHanaSR-angi takes place of the origianl one?

Reply:
ok